static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -25,6 +25,13 @@
         @include mq(desktop) {
             width: 100%;
         }
+
+        // hide * home links, which are duplicated on large screens
+        &:first-child {
```

Comment:
I think it's fine to go with this assumption, because this is only applied to items, which are in the second level of the primary links. These should always have a "home" link, otherwise it would be broken on small screens. We could add this to the "potentially refactor" along the attribute selectors at the end ...